docs/components/modeler/bpmn/timer-events/timer-events.md

```diff
@@ -79,11 +79,13 @@ If the duration is zero or negative, the timer fires immediately.
 
 A cycle defined as ISO 8601 repeating intervals format; it contains the duration and the number of repetitions. If the repetitions are not defined, the timer repeats infinitely until it is canceled.
 It's possible to define a start time. By doing this, the timer triggers for the first time on the given start time. Afterwards, it will follow the interval as usual.
+Additionally, you can specify a time cycle using cron expressions. Please see the [CronExpression Tutorial](https://spring.io/blog/2020/11/10/new-in-spring-5-3-improved-cron-expressions) for additional information about using cron expressions.
 
 - `R5/PT10S`: Every 10 seconds, up to five times
 - `R/P1D`: Every day, infinitely
 - `R/2022-01-01T10:00:00+02:00[Europe/Berlin]/P1D`: Every day infinitely, starting from January 1, 2022 at 10 a.m. UTC plus 2 hours
 - `R3/2022-04-27T17:20:00Z/P1D`: Every day up to three times, starting from April 27, 2022 at 5:20 p.m. UTC
+- `0 0 9-17 * * MON-FRI`: On the hour nine-to-five weekdays
```

Comment:
I would rephrase this to clarify. "Every hour from 9–5 p.m. (enter timezone) Monday–Friday."

Reply:
I agree, but would keep the part about `on the hour`.

Something like: `Every hour on the hour from 9-5 p.m. UTC Monday-Friday`